libs/core/src/lib/list/list-item/list-item.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="checkbox || radio">
-    <div fd-form-item class="fd-list__form-item">
+    <div fd-form-item class="fd-list__form-item" aria-hidden="true">
```

Comment:
If you add aria-hidden true to an element, the screen reader won't be able to read the element and it's children. Is it an expected behaviour to hide the checkbox and the radio from the SR?

<img width="889" alt="Screenshot 2023-04-12 at 6 03 58 PM" src="https://user-images.githubusercontent.com/39598672/231596854-a4ae60e3-c058-43ff-94c3-2f039404a6ca.png">